server/container/guice/guice-common/sample-configuration/james-database.properties

```diff
@@ -0,0 +1,40 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Yes, we can but it should duplicate more files (*.xml file). It's clear for one file but it will difficult for maintain with other files. WDYT?